File: output/ChatGPT/avatar/Python/Java/codeforces_92_A.java
import java.util.Scanner;

public class codeforces_92_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        int m = scanner.nextInt();
        int i = 1;
        
        while (m >= i) {
            m -= i;
            i = (i % n) + 1;
        }
        
        System.out.println(m);
    }
}

File: output/CodeGen/avatar/Java/Python/codeforces_92_A.py
def main(args):
    scn = Scanner(System.in)
    n = scn.nextInt()
    k = scn.nextInt()
    i = 1
    req = 1
    while k - req >= 0:
        if k - req >= 0:
            k = k - req
        else:
            break
        i += 1
        if i % n!= 0:
            req = i % n
        else:
            req = n
    print(k)